django/db/backends/base/schema.py

```diff
@@ -1686,9 +1705,11 @@ def _unique_sql(
         opclasses=None,
         expressions=None,
     ):
-        if (
-            deferrable
-            and not self.connection.features.supports_deferrable_unique_constraints
```

Comment:
This seems to have a small change in semantics, I think? Shall `condition`, `include` and `expressions` be passed as None to ensure the same semantics as before is preserved?

Reply:
It works exactly the same. This method calls `_create_unique_sql()`, when `condition`, `include`, or `expressions` are defined, that has exactly the same check.